cmd/devices-api/api.go

```diff
@@ -232,6 +233,7 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 	v1Auth.Post("/user/devices/fromvin", userDeviceController.RegisterDeviceForUserFromVIN)
 	v1Auth.Post("/user/devices/fromsmartcar", userDeviceController.RegisterDeviceForUserFromSmartcar)
 	v1Auth.Post("/user/devices", userDeviceController.RegisterDeviceForUser)
+	v1Auth.Post("/user/devices/:tokenID/credentials", userDeviceController.GetUserDevicesWithCredentials)
```

Comment:
I think this is a dangerous path—`user/devices/{something}` is already taken and it makes it seem like the token id will be for a device. We're really acting on the integration. Can we do `/v1/integration/2/credentials`?